x/distribution/client/rest/query.go

```diff
@@ -65,7 +65,24 @@ func registerQueryRoutes(cliCtx context.CLIContext, r *mux.Router, queryRoute st
 
 }
 
-// HTTP request handler to query the total rewards balance from all delegations
+// coinsReturn helps generate documentation for REST routes
+type coinsReturn struct { // nolint: deadcode unused
```

Comment:
let's move all swagger types to `swagger.go` with a grouped `type` clause that has a single lint statement (just copy-pasta from another swagger.go file).